src/main/java/pl/magisterka/dominikszojda/controler/DogController.java

```diff
@@ -1,17 +1,49 @@
 package pl.magisterka.dominikszojda.controler;
 
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 import pl.magisterka.dominikszojda.entity.DogEntity;
+import pl.magisterka.dominikszojda.mappers.DogMapper;
+import pl.magisterka.dominikszojda.request.DogCreateRequestDto;
+import pl.magisterka.dominikszojda.respons.DogResponseDto;
 import pl.magisterka.dominikszojda.service.DogService;
 
+import java.util.List;
+import java.util.UUID;
+
 @RequiredArgsConstructor
+@RestController
+@RequestMapping("dogs")
 public class DogController {
 
     private final DogService dogService;
+    private final DogMapper mapper;
+
+    @PostMapping("/add-dog")
+    public ResponseEntity<DogResponseDto> createDog(@RequestBody DogCreateRequestDto dogRequest) {
+        DogResponseDto responseEnt = dogService.createDog(dogRequest);
+
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(responseEnt);
+    }
+
+    @GetMapping("")
+    public ResponseEntity<List<DogEntity>> getDogs() {
 
-    private DogEntity addDog(){
-        return dogService.addDog();
+        List<DogEntity> responseEnt = dogService.getDogs();
+        return ResponseEntity
+                .status(HttpStatus.OK)
+                .body( responseEnt);
     }
 
+    @GetMapping("/{UUID}")
+    public ResponseEntity<DogResponseDto> getDogs(@PathVariable UUID UUID) {
```

Comment:
w Javie nazwy parametrów pisze się z małej litery
```suggestion
    public ResponseEntity<DogResponseDto> getDogs(@PathVariable UUID uuid) {
```

Reply:
poprawione